.travis.yml

```diff
@@ -63,9 +60,9 @@ matrix:
     - python: 3.5
 
 before_install:
-  - wget -O geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.12.0/geckodriver-v0.12.0-linux64.tar.gz
-  - gunzip -c geckodriver.tar.gz | tar xopf -
-  - chmod +x geckodriver && sudo mv geckodriver /usr/local/bin
+  - wget -O /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/releases/download/v0.12.0/geckodriver-v0.12.0-linux64.tar.gz
```

Comment:
Why cannot we download to the local dir? Or maybe even a sub directory which we could even add to the cache? We should also update to the 0.13.0 release of geckodriver.